jobs/build.Jenkinsfile

```diff
@@ -88,10 +88,13 @@ def strict_build_param = is_mechanical ? "" : "--strict"
 // workflow, only the qemu image is built).
 def cosa_memory_request_mb = 6.5 * 1024 as Integer
 
+// leave 512M for overhead; VMs are 1G each
+def ncpus = ((cosa_memory_request_mb - 512) / 1024) as Integer
+
 // the build pod runs most frequently and does the majority of the computation
 // so give it some healthy CPU shares
```

Comment:
maybe let's add a better comment here on why we're basing CPU requests on memory (which seems odd to me). basically what we're doing here is saying our limiting factor is memory and our VMs are approximately 1G memory per VM so request that many CPUs.

A more direct way of doing this may be defining first here how "parallel" we want our kola tests to run and then deriving both CPU and memory from that.

i.e. if we want a parallel of 6 (5 kola + 1 kola upgrade) then we define parallel of 6 and we need 6*1G+512M of memory and 6 cpus.

Reply:
Though technically currently we're defining the amount of memory we need based on xz, not based on kola parallelism. The latter is being derived from that. I've added a comment to expand on that.

I think we could flip it around and say kola tests is our most memory intensive factor though it's mostly playing with words unless we actually want to go past 6.